pkg/restapi/did/operation/operations.go

```diff
@@ -72,41 +79,66 @@ type httpClient interface {
 // GetRESTHandlers get all controller API handler available for this service
 func (o *Operation) GetRESTHandlers() []Handler {
 	return []Handler{
-		support.NewHTTPHandler(proxyURL, http.MethodGet, o.proxy),
+		support.NewHTTPHandler(resolveURL, http.MethodGet, o.resolve),
 	}
 }
 
-// Proxy will proxy requests based on provided configuration
-func (o *Operation) proxy(rw http.ResponseWriter, req *http.Request) {
+func (o *Operation) resolve(rw http.ResponseWriter, req *http.Request) {
 	did := mux.Vars(req)["did"]
 
-	log.Debugf("proxy received request for DID: %s", did)
+	log.Debugf("resolve received request for DID: %s", did)
 
-	destinationURL, err := o.ruleProvider.Transform(did)
+	result, err := o.ruleProvider.Resolve(did)
```

Comment:
Already here having Resolve is confusing - I would be better to change it to Match (or keep Transform)